client/js/lib/addon/loadAddonComponents.js

```diff
@@ -0,0 +1,34 @@
+import { checkResponse, dpRpc } from '@demos-europe/demosplan-ui'
+import Vue from 'vue'
+
+export default function loadAddonComponents (hookName) {
+  const params = {
+    hookName: hookName
+  }
+
+  const addons = []
+
+  return dpRpc('addons.assets.load', params)
+    .then(response => checkResponse(response))
+    .then(response => {
+      const result = response[0].result
```

Comment:
can't test it, but shouldn't be `const addons = []` within the `then` ?

And if it doesn't work accessing `Vue` here, we could pass it in the Method. But I think it will work like it is...

Reply:
this https://github.com/demos-europe/demosplan-core/pull/1486/commits/34048d60da081d14a985a4bd06fae3a6ab8eb573 is fixing the addon loading mechanism per se. However there is now a new demosplan-ui error...